CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 
 **ENHANCEMENTS**
 - Configure NFS threads to be max(8, num_cores) for performance
+- Disable X/GDM daemon on compute node if dcv is enabled
```

Comment:
Needs to be updated. I'd move this to the changes and write something like: 
Set default systemd runlevel to `multi-user.target` on all OSes. The runlevel is set to `graphical.target` only when DCV is enabled.

Reply:
Done